auto/sbt

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+cd $(dirname $0)/..
+
+# Clean up at the end
+trap "docker-compose down --volumes --remove-orphans" 0
+
+# Create volume
+docker volume create --name ivy-cache > /dev/null
+docker volume create --name docker-scala-sbt-cache > /dev/null
+
+# rebuild if necessary
+docker-compose build dev
+
+# Run dev script or use what is passed in as arguments
+docker-compose run --rm --service-ports dev "$@"
```

Comment:
A common pattern at REA is to name a script like this `auto/dev-environment`. Then you can make the `docker-compose` command `docker-compose run --rm --service-ports dev "${@:-bash}" so it'll run bash by default. Then you can create an `auto/sbt` script which does something like `auto/dev-environment sbt $@`. Then you could do as @sanjivsahayamrea suggested.

Reply:
Good idea @shaun-whitely. We do call it `dev-environment` in Listings.